crawl-ref/source/religion.cc

```diff
@@ -226,12 +251,18 @@ const vector<god_power> god_powers[NUM_GODS] =
 
     // Jiyva
     { { 1, ABIL_JIYVA_CALL_JELLY, "request a jelly" },
-      { 3, "Jiyva will mutate your body as you gain piety.",
-           "Jiyva will no longer mutate your body." },
+      { 3, "Jiyva will now mutate your body and modify your attributes as you gain piety.",
+           "Jiyva will no longer mutate your body and modify your attributes.",
+           "Jiyva will mutate your body and modify your attributes as you gain piety." },
+      { 3, "Jiyva is now protecting you from corrosive effects.",
+           "Jiyva will no longer protect you from corrosive effects.",
+           "Jiyva protects you from corrosive effects." },
       { 4, ABIL_JIYVA_SLIMIFY, "turn your foes to slime" },
       { 5, "You may now expel jellies when seriously injured.",
-           "You will no longer expel jellies when injured." },
-      { 5, ABIL_JIYVA_CURE_BAD_MUTATION, "call upon Jiyva to remove your harmful mutations" },
+           "You will no longer expel jellies when injured.",
```

Comment:
It looks like these are backwards for gain/loss.

Reply:
If you mean the "You may now expel jellies" lines, they're not:
`"You may now expel jellies when seriously injured"` is `gain`
`"You will no longer expel jellies when injured"` is `loss`
`"You may expel jellies when seriously injured"` is `general`, the text on the ^ screen.

Reply:
Oh, I read the diff wrong. My bad.